approach_data_structure_task.md

```diff
@@ -66,7 +65,7 @@ Question 1: Can one `article` be written in more than one `topic`? Yes, that is
 Question 2 (The Million Question): Can and should one single `article` refer to a million `topic`?
 This is where reality dictates the answer. While in theory an article could reference a million topics, we know that this won't be the case. A typical article (blog post) will probably have a single or few topics. So, it is safe to say, that based on the limitation which is derived from the fact we are building _real_ sites and not answering academic papers, the `article` refers to the `topic`.
 
-![](images/approach_data_structure_task/5a.jpg)
+![](images/approach_data_structure_task/articles-reference-topic.png)
```

Comment:
@OritGeron you missed one arrow from `article` to `author`

![wip__replace_sketches_in_approach_new_task_by_liatsade_ _pull_request__46_ _gizra_gizra-way-book](https://cloud.githubusercontent.com/assets/20393485/18626345/31f32844-7e5d-11e6-8c5b-c3d2d55ad416.jpg)